addons/l10n_fr/models/__init__.py

```diff
@@ -2,3 +2,4 @@
 from . import template_fr
 from . import res_partner
 from . import res_company
+from . import account_move
```

Comment:
technically these should be alphabetically sorted...
even though it won't be sorted, please put this line above